packages/rn-tester/js/utils/RNTesterList.ios.js

```diff
@@ -196,14 +196,12 @@ const Components: Array<RNTesterModuleInfo> = [
     category: 'Basic',
     supportsTVOS: true,
   },
-  /* [TODO(macOS GH#774) - NewArchitectureExample depends on Fabric, which we don't have on macOS yet
   {
     key: 'NewArchitectureExample',
     category: 'UI',
     module: require('../examples/NewArchitecture/NewArchitectureExample'),
     supportsTVOS: false,
   },
-  ]TODO(macOS GH#774) */
```

Comment:
@Saadnajmi Any ideas for conditionally rendering only when new arch is enabled? The `RNTesterLoadAllPages` tests are failing since due to a Redbox error when trying to render w/o new arch enabled.

![CleanShot 2022-11-28 at 10 22 55](https://user-images.githubusercontent.com/96719/204352134-782ea0e1-9bcc-4b94-8d0b-bd9860ca7d4e.jpg)

https://dev.azure.com/ms/react-native/_build/results?buildId=401553&view=logs&j=e300bc10-04fb-5308-1882-668349212529&t=4530efaa-9aa7-5589-6265-48e29ad5763e

Reply:
I'll try and look into it

Reply:
AFIK, there isn't an easy way to runtime check if fabric is enabled without hacks :/ 

You can try this: https://github.com/microsoft/react-native-test-app/blob/3c384cff862e4bfb084a0c534817db5703fbeac8/example/App.js#L159

Reply:
However.. if rn-tester now just depends on fabric, I wonder if we should be running our tests with paper at all. And afaik these tests are upstream so why don't they fail? Some more investigation I guess...

Reply:
Going with the "hack".